src/detect_breakends/junction_detection.cpp

```diff
@@ -245,7 +245,7 @@ void analyze_cigar(std::string chromosome,
  */
```

Comment:
This is a duplicate of the documentation in the associated header file. If I remember correctly, Jörg said that it's sufficient to have documentation in the header file.